### PR TITLE
Convert to origami 2.0 library

### PR DESCRIPTION
**Pull request by JakeChampion**
o-utils is not a component, it is two javascript utility functions. I've changed it to be a library instead of a component for that reason.

### CODE REVIEW

File: origami.json [hunk 1]
```diff
@@ -1,9 +1,6 @@
 {
-	"description": "Provides JS helper functions, such as throttling and debouncing",
-	"keywords": ["debounce", "throttle"],
-	"origamiType": "module",
-	"origamiCategory": "utilities",
-	"origamiVersion": 1,
+	"origamiType": "library",
```
Comment (chee) [inline origami.json:2]: :+1:

**Review by chee — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
+		"sinon": "^10.0.0"
 	},
 	"scripts": {
-		"start": "npx serve ./demos/local",
-		"build": "npm_config_yes=true npx \"origami-build-tools@^10\" install && npm_config_yes=true npx \"origami-build-tools@^10\" demo"
+		"test": "mocha test"
+	},
+	"volta": {
+		"node": "14.16.1",
+		"npm": "7.10.0"
```
Comment (chee) [inline package.json]: :+1:

File: package.json [hunk 3]
```diff
@@ -1,19 +1,36 @@
...
-		"remark-preset-lint-origami-component": "^1.1.3",
-		"stylelint": "^13.7.0",
-		"stylelint-config-origami-component": "^1.0.4"
+		"remark-preset-lint-origami-component": "prerelease",
+		"sinon": "^10.0.0"
 	},
 	"scripts": {
-		"start": "npx serve ./demos/local",
-		"build": "npm_config_yes=true npx \"origami-build-tools@^10\" install && npm_config_yes=true npx \"origami-build-tools@^10\" demo"
+		"test": "mocha test"
```
Comment (chee) [inline package.json:30]: :+1:

**Review by chee — APPROVED** [review]

**Review by chee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v2.0.0 :tada:

**Pull request merged**